demosplan/DemosPlanUserBundle/Logic/UserMapperDataportGatewayHH.php

```diff
@@ -631,14 +630,14 @@ protected function updateOrgaMode2(Orga $orga, $isToeb = false)
      * Create department.
      *
      * @param string $orgaGwId
```

Comment:
```suggestion
```

Reply:
What is your suggestion here @breierDemos ? I do not see any difference

Reply:
got it, the line was deleted. Done in https://github.com/demos-europe/demosplan-core/commit/43549e44a9dc4e761329fd2132b4fe6c9a151004